src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_utils.cpp

```diff
@@ -176,14 +169,22 @@ ov::snippets::lowered::ExpressionPtr get_copy_b_expr(const ov::snippets::lowered
 }  // namespace repacking
 }  // namespace intel_cpu::brgemm_utils
 
-template <>
-EnumNames<ov::intel_cpu::brgemm_utils::BRGEMM_TYPE>& EnumNames<ov::intel_cpu::brgemm_utils::BRGEMM_TYPE>::get() {
-    static auto enum_names = EnumNames<ov::intel_cpu::brgemm_utils::BRGEMM_TYPE>(
-        "ov::intel_cpu::jit_bgremm_utils::BRGEMM_TYPE",
-        {{"stand_alone", ov::intel_cpu::brgemm_utils::BRGEMM_TYPE::STAND_ALONE},
-         {"with_amx", ov::intel_cpu::brgemm_utils::BRGEMM_TYPE::WITH_AMX},
-         {"with_compensations", ov::intel_cpu::brgemm_utils::BRGEMM_TYPE::WITH_COMPENSATIONS},
-         {"repacking_only", ov::intel_cpu::brgemm_utils::BRGEMM_TYPE::REPACKING_ONLY}});
-    return enum_names;
+bool AttributeAdapter<ov::intel_cpu::brgemm_utils::BrgemmConfig>::visit_attributes(AttributeVisitor& visitor) {
+    bool with_wei_repacking = m_ref.with_wei_repacking();
+    bool with_comps = m_ref.with_compensations();
+    bool are_wei_blocked = m_ref.are_wei_blocked();
+    bool is_amx = m_ref.is_amx();
+    std::string isa = JIT_IMPL_NAME_HELPER("", m_ref.isa(), "");
+    size_t wei_n_blk = m_ref.wei_n_blk();
+    size_t wei_k_blk = m_ref.wei_k_blk();
+
+    visitor.on_attribute("with_brgemm_copy_b", with_wei_repacking);
+    visitor.on_attribute("with_compensations", with_comps);
+    visitor.on_attribute("are_wei_blocked", are_wei_blocked);
```

Comment:
Should `are_wei_constant` be also placed here?

Reply:
Good catch, thanks!